### PR TITLE
util: remove some unused code in admin package

### PR DESCRIPTION
**Pull request by wjhuang2016**
### What problem does this PR solve? 
There are some functions used only in test file, remove them and these tests.

### Check List 

Tests 

 - No code

Code changes

 - Has exported function/method change

Side effects

Related changes

Release note

### CODE REVIEW

File: util/admin/admin_test.go [hunk 1]
```diff
@@ -226,7 +214,7 @@ func (s *testSuite) TestCancelJobs(c *C) {
 	errs, err = CancelJobs(txn, []int64{-1})
 	c.Assert(err, IsNil)
 	c.Assert(errs[0], NotNil)
-	c.Assert(errs[0].Error(), Equals, "[admin:4]DDL Job:-1 not found")
+	c.Assert(errs[0].Error(), Matches, "*DDL Job:-1 not found")
```
Comment (Deardrops) [inline util/admin/admin_test.go]:
```suggestion
	c.Assert(ErrDDLJobNotFound.Equal(errs[0])), IsTrue)
```

Comment (wjhuang2016) [reply util/admin/admin_test.go]: `c.Assert(errs[0].Error(), Matches, "*DDL Job:-1 not found") ` also check the job ID， I think we should keep it.

**Pull request merged**